### ISSUE TITLE
Tachiyomi deletes read status of manga

### ISSUE THREAD

**Issue by RocGoddess** [tachiyomiorg/tachiyomi#6513]
### Steps to reproduce

Open manga from library.

### Expected behavior

Previously read chapter should be greyed out showing they have been read.

### Actual behavior

Previously read chapters are not greyed out.

### Tachiyomi version

0.12.3

### Android version

Android 11

### Device

Galaxy A71

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by curche** [comment]
Kindly provide more details. 
 - When does the chapter get marked read?
 - Are they always marked unread or do these go unread after a Library Global Update?
 - Do all the manga have this problem or does this happen to only some manga? In particular, are these manga from MangaOwl?

**Comment by RocGoddess** [comment]
Hi sorry for not giving more details. I didn't know what extra details were needed.
1. I had read the chapters a while back and was trying to continue from where I had stopped.
2. It seems to randomly happen but only after a library update. 
3. Most of them where from mangaowl but one was from mangadex.
4. I also lost my bookmarks as well (all mangaowl) but I don't know if that's important.

**Comment by arkon** [comment]
MangaOwl is broken and chapters are constantly being replaced as new ones. This can also occur in other sources, but it's expected behaviour if the app considers them as new.
